demo/chromecast/chromecast-player.js

```diff
@@ -16,41 +16,60 @@ import {EmbedPlayer, ChromecastControls} from '../../dist/bundle.js';
     const continueFromPreviousPosition = urlParams.get('continueFromPreviousPosition');
     const tokenParameter = token ? {token} : {};
 
+    const splashEl = document.querySelector('.splash-overlay');
+    const metaEl = document.querySelector('.meta');
+
     const initParam = {
         selector: '.video-wrapper',
         options: {
             autoplay: autoplay && autoplay === 'true',
-        }
+        },
+        overlayElement: metaEl,
     };
     if (posterImageUrl) {
         initParam.options.poster = posterImageUrl;
     }
 
-    const player = new EmbedPlayer({projectId, apiBaseUrl, chromecastReceiverAppId});
+    const embedPlayer = new EmbedPlayer({projectId, apiBaseUrl, chromecastReceiverAppId});
 
     const containerEl = document.getElementById('buttons-container');
     document.getElementById('video-button-start').addEventListener('click', playVideo);
-    document.getElementById('video-button-stop').addEventListener('click', stopCastVideo);
     document.getElementById('video-button-destroy').addEventListener('click', destroyVideo);
 
-    player.initChromecast().then(() => {
-        const controls = new ChromecastControls(player.getCastPlayer(), player.getCastPlayerController());
-        document.getElementById('cast-wrapper').style.visibility = 'visible';
+    if (posterImageUrl) {
+        splashEl.style.backgroundImage = `url(${posterImageUrl})`;
```

Comment:
When there is no image, bg is transparent and metas can become invisible. 
Can we add fallback background (e.g. 0,0,0,0.3) when there is no image?